Sources/Parsing/Parsers/StartsWith.swift

```diff
@@ -1,4 +1,24 @@
 /// A parser that parses a sequence of elements from its input.
+///
+/// This parser is named after `Sequence.starts(with:)`, and tests that the input it is parsing
+/// starts with a given prefix by calling this method under the hood.
```

Comment:
How about this to not further confuse this operator with the `prefix` operator?

```suggestion
/// starts with a given subsequence by calling this method under the hood.
```